doc/src/MainPage.dox

```diff
@@ -231,16 +228,16 @@
  * OpenAssetIO was developed to make it possible for an
  * @ref asset_management_system to support any @ref host application
  * without specific knowledge, and vice versa. This is achieved though
- * the use of common specifications and descriptions and two simple
- * rules for the manager:
+ * the use of common specifications that define well-known traits and
+ * a two simple rules for the manager:
  *
- * - Store and recall a @ref primary_string
- * - Allow filtering by @ref Specification
+ * - Store and recall the set of traits, and their properties registered
+ *   for any given entity.
```

Comment:
[consider]
```suggestion
 * - Store and recall the set of traits and their properties for
 *   any given entity.
```

Reply:
I had intentionally left `registered` in to try and hint at the obligation to store what is published, but can remove if you think it's clearer? Minor rewording [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/327/commits/d75cb988712864a1520b938cf69710e6881c2eff).

Reply:
Reworded version looks good :+1: